docs/docs/build/olap/olap.md

```diff
@@ -60,4 +61,13 @@ When ClickHouse has been configured as the [default OLAP engine](../../reference
 
 <div className="center-content">
 ![External ClickHouse tables](/img/build/connect/external-tables/external-clickhouse-table.png)
-</div>
\ No newline at end of file
+</div>
+
+## Pinot
+
+When Pinot has been configured as the [default OLAP engine](../../reference/project-files/rill-yaml.md#configuring-the-default-olap-engine) for your project, 
+any existing external tables that Rill can read and query should be shown through the Rill Developer UI under `Tables` section in left pane. You can then create dashboards using these external Pinot tables.
```

Comment:
I added a new line for readability when its rendered it will be all in single paragraph w/o new line

Reply:
Unless the line width is very small, it will just break weirdly in other IDEs/interfaces of a different size – easier to let the editor break (also what most other docs pages do). Here is what it looked like when I reviewed:

<img width="1072" alt="Screenshot 2024-04-24 at 11 42 28" src="https://github.com/rilldata/rill/assets/791356/6831661e-224c-43e0-b1e2-e4ea33e5a34a">